### PR TITLE
feat(sdk): add screen to manage service account and API keys

### PR DESCRIPTION
**Pull request by rsbh**

### CODE REVIEW

File: sdks/js/packages/core/react/components/organization/api-keys/index.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  return (
+    <EmptyState
+      icon={<ExclamationTriangleIcon />}
+      heading="Restricted Access"
+      subHeading={`Admin access required, please reach out to your admin incase you want to generate a key.`}
+    />
+  );
+};
+
+const useAccess = (orgId?: string) => {
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/index.tsx:52]: As its a hook, should be move it to hooks folder?

File: sdks/js/packages/core/react/components/organization/api-keys/index.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import { Flex, Text, EmptyState, Button } from '@raystack/apsara/v1';
+import styles from './styles.module.css';
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/index.tsx:2]: CSS can be placed in the end?

File: sdks/js/packages/core/react/components/organization/api-keys/index.tsx (hunk 1 above)
Comment (rsbh) [reply sdks/js/packages/core/react/components/organization/api-keys/index.tsx:52]: The access logic is local to the component. Once we make it generic and start using it in another place, then we can move it to a separate file.

File: sdks/js/packages/core/react/components/organization/api-keys/index.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  );
+};
+
+const useAccess = (orgId?: string) => {
+  const resource = `app/organization:${orgId}`;
+  const listOfPermissionsToCheck = useMemo(() => {
+    return [
+      {
+        permission: PERMISSIONS.UpdatePermission,
+        resource: resource
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/index.tsx:58]:
```suggestion
        resource
```

**Review by paanSinghCoder — APPROVED** [review]

File: sdks/js/packages/core/react/components/organization/api-keys/index.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      icon={<ExclamationTriangleIcon />}
+      heading="Restricted Access"
+      subHeading={`Admin access required, please reach out to your admin incase you want to generate a key.`}
+    />
+  );
+};
+
+const useAccess = (orgId?: string) => {
+  const resource = `app/organization:${orgId}`;
+  const listOfPermissionsToCheck = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (rsbh) [reply sdks/js/packages/core/react/components/organization/api-keys/index.tsx:54]: The `listOfPermissionsToCheck` is a dependency on the `usePermissions` hook. It is an array of objects that give different memory locations and thus do unnecessary render to the `usePermissions` hook

**Pull request merged**